selfdrive/car/subaru/interface.py

```diff
@@ -40,43 +40,32 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     else:
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
+    ret.mass = candidate.config.specs.mass
```

Comment:
move here? https://github.com/commaai/openpilot/blob/015eed5d61c583328d419a7990ffac0eee97edd4/selfdrive/car/interfaces.py#L110

Reply:
once all brands are done?

Reply:
why not now? just check if it’s none?